app/javascript/article-form/articleForm.jsx

```diff
@@ -476,7 +477,8 @@ export class ArticleForm extends Component {
 
         <KeyboardShortcuts
           shortcuts={{
-            'ctrl+shift+KeyP': this.fetchPreview,
+            [`${Runtime.currentOS() === 'macOS' ? 'cmd' : 'ctrl'}+shift+KeyP`]:
```

Comment:
Should this use `const keyboardShortcutModifier` from a later definition?

Reply:
+1. It makes sense to consolidate this and import it in the places it's used.

Reply:
Yeah good point, otherwise we will have this same ternary littered everywhere 😄  I've extracted this into a helper in the `Runtime` class